Makefile

```diff
@@ -45,36 +45,39 @@ SRC = src/debug/main_a.c\
 		src/exec/pipe_utils.c\
 		src/exec/redirects_utils.c
 
-OBJ = $(SRC:.c=.o)
+OBJ = $(SRC)
+OBJ := $(OBJ:%.c=%.o)
 
-HEADER = incl/minishell.h
+HEADER = -I ./incl/
 
 CFLAGS = -Wall -Werror -Wextra -g
 
-GCC = gcc
+CC = cc
```

Comment:
Changed how we compile quite a bit: gcc wouldn't compile the program when the global variable (g_signal) was properly defined and the subject says that we should be actually using cc. Therefore, everything is rewritten for that and now works on the school PCs and mine :)

The main difference is in how the object files are created. Previously, they were created on one compilation line (line 69 in the old version). Now, they are first created separately (lines 62-63) and then linked together in line 71. This required changing also how libftprintf is added to the compilation.